src/scripts.js

```diff
@@ -17,42 +13,34 @@ const searchBar = document.getElementById('searchBar');
 const navMyRecipes = document.getElementById('navMyRecipes');
 const navUserInfo = document.getElementById('navUserInfo');
 const logo = document.getElementById('logo');
-
-const recipeRepo = new RecipeRepository(recipeData);
+let users;
+let ingredients;
+let recipes;
+let recipeRepo;
 let modalRecipe;
 let currentUser;
 let savedView = false;
+let currentView;
```

Comment:
S/o @qrispi for this - brilliant solution!

Reply:
This was very creative! Looks much cleaner.